pacai/ui/pacmanDisplay.py

```diff
@@ -0,0 +1,663 @@
+import math
+import io
+
+from abc import ABC
+from abc import abstractmethod
+from pacai.ui import graphicsUtils
+from pacai.ui import graphicsConstants
+from pacai.core.game import Directions
```

Comment:
Pacai.core.game should be before pacai.ui

Reply:
There is no Directions in pacai.ui. Did you mean pacai.core.directions?

Reply:
Nonetheless, the linter said that Directions is imported but never used so I removed it.